src/EscapeAnalysis.jl

```diff
@@ -540,25 +540,139 @@ isaliased(x::Union{Argument,SSAValue}, y::Union{Argument,SSAValue}, estate::Esca
 isaliased(xidx::Int, yidx::Int, estate::EscapeState) =
     in_same_set(estate.aliasset, xidx, yidx)
 
-struct ArgEscapeInfo
-    EscapeBits::UInt8
+struct InterEscapeInfo
```

Comment:
It feels like there should be a way for us to re-use code from `EscapeInfo` here, but maybe not possible with how subtyping is currently supported, and ultimately not super important